searx/webapp.py

```diff
@@ -413,6 +413,8 @@ def render(template_name, override_theme=None, **kwargs):
 
     kwargs['brand'] = brand
 
+    kwargs['contact_url'] = settings.get('general').get('contact_url')
```

Comment:
If we ship CONTACT_URL within the `brand`  namespace, this line can be dropped.

Reply:
Ok, I made 65af3f3 now.